_config.yml

```diff
@@ -8,8 +8,8 @@ description: Java Design Patterns
 
 # ! Attention: the lines starting with "baseurl" and "url" are edited on travis build with a search and replace regex
 # Any Change in those two lines should be checked with the "cibuild" script
-baseurl: "/java-design-patterns" # the subpath of your site, e.g. /blog/
-url: "https://iluwatar.github.io" # the base hostname & protocol for your site
+baseurl: "" # the subpath of your site, e.g. /blog/
+url: "http://localhost:4000" # the base hostname & protocol for your site
```

Comment:
@markusmo3 
My mistakes I have not reverted my local changes in the config file.
Reagarding the change, it was mentioned at [jekyll kick start](https://github.com/iluwatar/java-design-patterns/wiki/07.-Working-with-the-web-site) point 6. So I have changed it locally.

I will revert the changes and commit again.